bot/exts/info/tags.py

```diff
@@ -138,6 +140,15 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.tags: dict[TagIdentifier, Tag] = {}
         self.initialize_tags()
+        self.bot.tree.copy_global_to(guild=discord.Object(id=GUILD_ID))
+
+    tag_group = app_commands.Group(name="tag", description="...")
+    # search_tag = app_commands.Group(name="search", description="...", parent=tag_group)
```

Comment:
```suggestion
```
Can this be removed?

Reply:
@Ibrahim2750mi I don't see this as resolved, the commented line is still there